subsys/fs/Kconfig.littlefs

```diff
@@ -104,4 +104,11 @@ config FS_LITTLEFS_HEAP_PER_ALLOC_OVERHEAD_SIZE
 
 endif # FS_LITTLEFS_FC_HEAP_SIZE <= 0
 
+config FS_LITTLEFS_BLK_DEV
+	bool "Enable support for littlefs on block devices"
+	default n
```

Comment:
n is default, the line is not needed.

Reply:
@de-nordic IIRC - checkpatch was complaining about missing "default n".

Reply:
Hm, strange. Need to check up on my active PR's.